Automattic/vip-cli
# Allow user to run multisite import even if wpSites.nodes doesn't exist.

## Description

If for some reason the sites list comes back undefined, we may still want the user to continue the import rather than halting it all together.

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Change this to return undefined: https://github.com/Automattic/vip/blob/master/src/bin/vip-import-sql.js#L293
1. Run `npm run build`
1. Run `../dist/bin/vip-import-sql.js @id <file.sql>`
1. Verify warning about no site is displayed
